src/ray/object_manager/pull_manager.cc

```diff
@@ -504,7 +509,7 @@ void PullManager::TryToMakeObjectLocal(const ObjectID &object_id) {
   }
 
   // TODO(ekl) should we more directly mark the object as lost in this case?
-  RAY_LOG(DEBUG) << "Object neither in memory nor external storage " << object_id.Hex();
+  RAY_LOG(INFO) << "Object neither in memory nor external storage " << object_id.Hex();
```

Comment:
Change these back to DEBUG?

Reply:
Oops, my bad! Yeah I was trying to debug a unit test.